src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -59,11 +62,11 @@ const RewardPanel = ({
   stepCount
 }: RewardPanelProps) => {
   const wm = useWithMobileStyle(styles.mobile)
-  const userChallenges = useSelector(getUserChallenges)
+  const userChallenges = useSelector(getOptimisticUserChallenges)
```

Comment:
Could also have a second version of this selector that takes in an id, and calls this one and indexes into it

Reply:
yeah, we had something similar before in the hooks, but it wasn't getting used so I figured i'd not bring it over